examples/book-and-change.py

```diff
@@ -20,22 +20,21 @@
         .execute()
     )
 
-    print("Created offer request: %s" % (offer_request.id))
+    print(f"Created offer request: {offer_request.id}")
 
     offers = client.offers.list(offer_request.id)
     offers_list = list(enumerate(offers))
 
-    print("Got %d offers" % len(offers_list))
+    print(f"Got {len(offers_list)} offers")
 
     selected_offer = offers_list[0][1]
 
-    print("Selected offer %s to book" % (selected_offer.id))
+    print(f"Selected offer {selected_offer.id} to book")
 
     priced_offer = client.offers.get(selected_offer.id)
 
     print(
-        "The final price for offer %s is %s (%s)"
-        % (priced_offer.id, priced_offer.total_amount, priced_offer.total_currency)
+        f"The final price for offer {priced_offer.id} is {priced_offer.total_amount} ({priced_offer.total_currency})"
```

Comment:
Not sure what it was before, but having read the comments pre-merge, thought I'd point out you can split a string across multiple lines as long as it's in parentheses (or uses a `\` at the end of the line) - i.e. this:

```python
    print(
        f"The final price for offer {priced_offer.id} is {priced_offer.total_amount} ({priced_offer.total_currency})"
    )
```

could become this:

```python
    print(
        f"The final price for offer {priced_offer.id} is {priced_offer.total_amount} "
        f"({priced_offer.total_currency})"
    )
```

or this, if you wanted to assign a long string to a variable (just for illustration):

```python
    msg = (
        f"The final price for offer {priced_offer.id} is {priced_offer.total_amount} "
        f"({priced_offer.total_currency})"
    ) # not a tuple!

    print(msg)
```

Note that the above doesn't construct a tuple - this is why tuples require a trailing comma when constructing via `()` syntax, e.g.

```python
>>> t = (1,)
>>> type(t)
<class 'tuple'>
>>> t = (1)
>>> type(t)
<class 'int'>
```

Reply:
You can, thanks for pointing this usage out, which is what I'd done. I decided that it wasn't worth the cost in readability for the adherence to PEP 8's default E501 so decided to exclude the examples from that check instead.